### ISSUE TITLE
Support for heredocs in Dockerfiles

### ISSUE THREAD

**Issue by sotoiwa** [hadolint/hadolint#682]
- [ ] This is a bug report
- [x] This is a feature request
- [ ] I searched existing issues before opening this one

### Expected behavior

Since Docker has introduced the new heredocs feature, we need support for this feature.

- [Introduction to heredocs in Dockerfiles](https://www.docker.com/blog/introduction-to-heredocs-in-dockerfiles/)

### Actual behavior

Hadolint cannot inspect Dockerfiles that use heredocs.

### Steps to reproduce the behavior

Example Dockerfile using heredoc:

```Dockerfiles
# syntax=docker/dockerfile:1.3-labs
FROM python:3-slim

RUN <<EOF
    apt-get update
    apt-get -y upgrade
    apt-get clean
    rm -rf /var/lib/apt/lists/*
EOF

WORKDIR /usr/src/app

RUN useradd -u 1001 default

COPY requirements.txt ./
RUN pip install --no-cache-dir -r requirements.txt

COPY app.py ./

RUN find / -xdev -perm /6000 -type f -exec chmod a-s {} \; || true

USER 1001

ENV PYTHONUNBUFFERED 1
ENTRYPOINT [ "python", "./app.py" ]
```

Results from hadolint:

```bash
$ hadolint --version
Haskell Dockerfile Linter 2.6.1-no-git
$ Hadolint Dockerfile 
Dockerfile:5:5 unexpected 'a' expecting '#', '\', ADD, ARG, CMD, COPY, ENTRYPOINT, ENV, EXPOSE, FROM, HEALTHCHECK, LABEL, MAINTAINER, ONBUILD, RUN, SHELL, STOPSIGNAL, USER, VOLUME, WORKDIR, a pragma, at least one space, or end of input 
```

**Comment by lorenzo** [comment]
that was fast, only a few days after being released as a feature officially :D
